### PR TITLE
Feature/gitflow

### PR DESCRIPTION
**Pull request by joseph-brennan**
our first sprint attempt for git flow

### CODE REVIEW

File: GitFlowHelp.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Smoke Git Procedures
+
+This is a rundown of how contributors working on the Smoke project should handle their git repositories, if you need a more in-depth review of Git Basics as well as Git-Flow basics please refer to https://github.com/joseph-brennan/smoke/blob/feature/gitflow/GitProcedures.md.
```
Comment (LSmith-Zenoscave) [inline GitFlowHelp.md]: use a relative link for the url. This should not point to any one fork.

**Comment by LSmith-Zenoscave** [comment]
This should be put in a `.github` folder and be named CONTRIBUTING.md
Please make sure to also follow up on the comments in the file.

File: GitFlowHelp.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# Smoke Git Procedures
```
Comment (LSmith-Zenoscave) [inline GitFlowHelp.md]: This should be something to the effect of "Contributions for smoke through git" but simpler.

File: GitFlowHelp.md (hunk 1 above)
Comment (Rflageolle) [reply GitFlowHelp.md]: Should we instead remove everything after "git repositories" as we are assuming base knowledge of git.

**Pull request closed without merging**